client/scripts/views/pages/subscriptions/chainEventSettings.ts

```diff
@@ -72,12 +71,48 @@ const EventSubscriptionRow: m.Component<IEventSubscriptionRowAttrs, {}> = {
   }
 };
 
+const IndividualEventSubscriptions: m.Component<{}, {
+  chain: string;
+  eventKinds: IChainEventKind[];
+  allSupportedChains: string[];
```

Comment:
what is this used for? it appears to only be checked once, and never assigned to.

Reply:
So `IndividualEventSubscriptions` was a component to test specific chain events against their specific subscription. It's not used in production, but I know jake was using it to debug chain events/notifications, so i left it in.

Reply:
@jnaviask thoughts on whether this is still necessary?

Reply:
could we add a comment if it's unused?

Reply:
Added a comment!